discovery-client/src/main/java/io/micronaut/discovery/aws/route53/registration/ServiceRegistrationStatusJob.java

```diff
@@ -0,0 +1,56 @@
+package io.micronaut.discovery.aws.route53.registration;
+
+import com.amazonaws.services.servicediscovery.model.GetOperationRequest;
+import com.amazonaws.services.servicediscovery.model.GetOperationResult;
+import io.micronaut.context.annotation.Value;
+import io.micronaut.discovery.aws.route53.Route53AutoRegistrationConfiguration;
+import io.micronaut.runtime.server.EmbeddedServerInstance;
+import io.micronaut.scheduling.annotation.Scheduled;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.io.*;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@Singleton
+public class ServiceRegistrationStatusJob {
+    private static final Logger LOG = LoggerFactory.getLogger(ServiceRegistrationStatusJob.class);
+
+    @Value("${server.serviceRegisterFile:/tmp/serviceOperationStatus.txt}")
+    String operationStatusFile;
+
+    @Inject
+    Route53AutoNamingRegistrationClient route53AutoNamingRegistrationClient;
+
+    @Inject
+    Route53AutoRegistrationConfiguration route53AutoRegistrationConfiguration;
```

Comment:
Could you change this to use constructor injection instead. Field injection is not really recommended.